os.go

```diff
@@ -164,6 +165,8 @@ func setDefaults() {
 
 	gOpts.cmds["doc"] = &execExpr{"$", `"$lf" -doc | $PAGER`}
 	gOpts.keys["<f-1>"] = &callExpr{"doc", nil, 1}
+
+	gOpts.fileinfofmt = "\033[36m%p\033[0m %c %u %g %s %t %L"
```

Comment:
If you want to make the permissions cyan by default, you should have a screenshot and a mention of the fact that you're changing the default in the description.

Reply:
Thanks I forgot to mention this, I updated the PR description.